games-action/supertuxkart/supertuxkart-1.0.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils desktop gnome2-utils
+
+DESCRIPTION="A kart racing game starring Tux, the linux penguin (TuxKart fork)"
+HOMEPAGE="https://supertuxkart.net/"
+SRC_URI="mirror://sourceforge/${PN}/SuperTuxKart/${PV}/${P}-src.tar.xz
+	mirror://gentoo/${PN}.png"
+
+LICENSE="GPL-2 GPL-3 CC-BY-SA-3.0 CC-BY-2.0 public-domain ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug fribidi +nettle openssl recorder wiimote"
+REQUIRED_USE="^^ ( nettle openssl )"
+
+# don't unbundle irrlicht and bullet
+# both are modified and system versions will break the game
+# https://sourceforge.net/p/irrlicht/feature-requests/138/
+
+RDEPEND="
+	dev-libs/angelscript
+	media-libs/glew:0=
+	media-libs/libpng:0=
+	media-libs/libvorbis
+	media-libs/openal
+	net-libs/enet:1.3=
+	net-misc/curl
+	sys-libs/zlib
+	virtual/glu
+	virtual/jpeg:0
+	virtual/libintl
+	virtual/opengl
+	x11-libs/libX11
+	x11-libs/libXxf86vm
+	fribidi? ( dev-libs/fribidi )
+	nettle? ( dev-libs/nettle )
+	openssl? ( dev-libs/openssl:0= )
+	recorder? ( media-libs/libopenglrecorder )
+	wiimote? ( net-wireless/bluez )"
+DEPEND="${RDEPEND}
+	sys-devel/gettext
```

Comment:
With EAPI-7, this and pkgconfig are BDEPEND. So something like:
```
RDEPEND="
	dev-libs/angelscript
	media-libs/glew:0=
	media-libs/libpng:0=
	media-libs/libvorbis
	media-libs/openal
	net-libs/enet:1.3=
	net-misc/curl
	sys-libs/zlib
	virtual/glu
	virtual/jpeg:0
	virtual/libintl
	virtual/opengl
	x11-libs/libX11
	x11-libs/libXxf86vm
	fribidi? ( dev-libs/fribidi )
	nettle? ( dev-libs/nettle )
	openssl? ( dev-libs/openssl:0= )
	recorder? ( media-libs/libopenglrecorder )
	wiimote? ( net-wireless/bluez )"
DEPEND="${RDEPEND}"
BDEPEND="
	sys-devel/gettext
	virtual/pkgconfig"
```

zlib is unnecessary dependency as its part of every system profile.

Reply:
No it isn't, at least not directly.